aten/src/ATen/EmptyTensor.cpp

```diff
@@ -160,7 +160,7 @@ SymInt computeStorageNbytes(
   // of the last element according to stride
   SymInt size = 1;
   for (const auto i : c10::irange(sizes.size())) {
-    if (TORCH_GUARD_SIZE_OBLIVIOUS(sizes[i].sym_eq(0))) {
+    if (TORCH_GUARD_OR_FALSE(sizes[i].sym_eq(0))) {
```

Comment:
I wonder if the proper thing to in this case is to completely remove the condition 
and at line 167 
do 
size += strides[i] * sym_max((sizes[i] - 1), 0);
or 
size += strides[i] * (sym_max(sizes[i],1) - 1);